amplify/data/resource.ts

```diff
@@ -23,6 +23,7 @@ const schema = a
         role: a.string().default("Participant"),
         email: a.string(),
         meals: a.boolean(),
+        attended: a.boolean().default("false"),
```

Comment:
Instead of attended, the field we want to target is the checkedIn field. If you could just change the files to make the change to that field, and then change the name of the other file to `setUserAsCheckedIn`, this can be merged right away.